src/v/cluster/tx_gateway_frontend.cc

```diff
@@ -279,9 +279,12 @@ ss::future<try_abort_reply> tx_gateway_frontend::do_try_abort(
   model::tx_seq tx_seq,
   model::timeout_clock::duration timeout) {
     auto maybe_tx = co_await stm->get_actual_tx(transactional_id);
-    if (!maybe_tx) {
-        // unknown tx => state was lost => can't be comitted => aborted
-        co_return try_abort_reply{.aborted = true, .ec = tx_errc::none};
+    if (!maybe_tx.has_value()) {
+        if (maybe_tx.error() == tm_stm::op_status::not_found) {
+            // unknown tx => state was lost => can't be comitted => aborted
```

Comment:
the conditional expression is on `not_found` error, but the comment is specific about `unknown` which is also a error code. is there a mistake here?

Reply:
it's wrong comment, I'm updated it